app/controllers/assignment_controller.rb

```diff
@@ -38,7 +38,7 @@ def loading_question_and_assignment
     @question = Pq.find_by(uin: params[:uin])
 
     if assignment_id
-      @assignment = @question.action_officers_pqs.find(assignment_id)
```

Comment:
Can revert to scoped query now (was not the cause of bug as initially suspected)